pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+This PHEP makes reference to "PyHC leadership": individuals or groups with statutory authority over the PyHC project. This may be the Principal Investigator or other individuals or groups as appropriately designated in the future, such as a steering committee.
+
+# PHEP Types
+There are three kinds of PHEP:
+
+1. A **Standards** Track PHEP describes a new standard for PyHC packages.
+2. An **Informational** PHEP describes a design issue, or provides general guidelines or information to the community, but does not propose a new standard. Informational PHEPs do not necessarily represent a community recommendation, so users and implementers are free to ignore Informational PHEPs or follow their advice.
+3. A **Process** PHEP describes a process surrounding PyHC, or proposes a change to (or an event in) a process. Process PHEPs are like Standards Track PHEPs but apply to areas other than PyHC packages. They may propose an implementation; they often require community consensus; unlike Informational PHEPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in PyHC development or other community activities. Any meta-PHEP is also considered a Process PHEP.
+
+# PHEP Applicability
+**Standards** Track PHEPs are recommended standards for PyHC packages. Most packages should make a good-faith effort to comply. PyHC core packages are strongly recommended to comply and generally should do so, absent a compelling reason not to. A PHEP which core packages cannot agree to implement likely does not have sufficient community support for approval. Standards track PHEP compliance should be considered as a factor in evaluating current and potential PyHC packages.
+
+**Informational** PHEPs are for the benefit of PyHC developers and users and may represent "best practices". They are not binding.
+
+**Process** PHEPs determine the process by which the PyHC community makes formal decisions and as such bind all participants in PyHC.
+
+For all three kinds, key words in PHEPs must be interpreted according to [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119); the use of `must`, `must not`, `should`, and `should not` are preferred to the alternatives.
+
+# PHEP Workflow
+## PHEP Editors
+<a name="phep-editors"></a>
+The PHEP editors are individuals responsible for managing the administrative and editorial aspects of the PHEP workflow (e.g. assigning PHEP numbers and changing their status). See [PHEP Editor Responsibilities & Workflow](#phep-editor-responsibilities-workflow) for details.
+
+PHEP editorship is by nomination of the current editors, subject to approval of PyHC leadership. In the event there are fewer than two active editors, PyHC leadership may appoint editors from active participants in the community, sufficient to bring the total to two.
```

Comment:
I like this framing. It's always great for me when the community does stuff instead of me/Julie doing it, so I encourage that, but it's not always how things go, so the "out" is warranted. And two seems like a good minimum.